src/Avalonia.Base/Rendering/Composition/Server/ServerContainerVisual.cs

```diff
@@ -0,0 +1,44 @@
+using System.Numerics;
+using Avalonia.Platform;
+
+namespace Avalonia.Rendering.Composition.Server
+{
+    /// <summary>
+    /// Server-side counterpart of <see cref="CompositionContainerVisual"/>.
+    /// Mostly propagates update and render calls, but is also responsible
+    /// for updating adorners in deferred manner
+    /// </summary>
+    internal partial class ServerCompositionContainerVisual : ServerCompositionVisual
```

Comment:
Not: filename doesn't match class name here either. Probably a few other places too.